bazel-mode.el

```diff
@@ -21,10 +21,23 @@
 ;;
 ;;; Code:
 
+(defconst bazel-mode-syntax-table
+  (let ((table (make-syntax-table)))
+	;; single and multiple line comment start
+	(modify-syntax-entry ?# "<" table)
+	;; single line comment end
+	(modify-syntax-entry ?\n ">" table)
+	table)
+  "Syntax table for `bazel-mode'.")
+
 ;;;###autoload
 (define-derived-mode bazel-mode prog-mode "Bazel"
-  "Major mode for editing Bazel BUILD and WORKSPACE files.")
-
+	"Major mode for editing Bazel BUILD and WORKSPACE files."
+	(setq comment-start "# ")
+	(setq comment-start-skip "#+")
+	(setq comment-end "")
+	(setq comment-use-syntax t)
+	(font-lock-ensure))
```

Comment:
I think the issue here is that font locking only works if `font-lock-defaults` is not nil. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Font-Lock-Basics.html. So you should set `font-lock-defaults` here, probably to `(nil)` since you don't have keywords yet.

Reply:
Yep that seems to work. Also thanks for the pointer.